packages/bpk-docs/src/pages/AnimationPage/AnimationPage.js

```diff
@@ -20,13 +20,19 @@ import React from 'react';
 import TOKENS from 'bpk-tokens/tokens/base.raw.json';
 import IOS_TOKENS from 'bpk-tokens/tokens/base.raw.ios.json';
 import ANDROID_TOKENS from 'bpk-tokens/tokens/base.raw.android.json';
+import IntroBlurb from './../../components/neo/IntroBlurb';
 
 import DocsPageBuilder from './../../components/DocsPageBuilder';
 import { getPlatformTokens } from './../../helpers/tokens-helper';
 
+const blurb = [
+  <IntroBlurb>Animation values for use in web, iOS and Android.</IntroBlurb>,
```

Comment:
On the other pages it says _in web, iOS and Android projects_ but on here it doesn't say _projects_. Maybe we should add that here for consistency.